korgan/legal_calc.py

```
@@ -2,27 +2,47 @@
+turn into a wrong amount in a court filing.
 """
 
 from __future__ import annotations
 
 import re
 from dataclasses import dataclass
 from datetime import date
+from decimal import Decimal, ROUND_HALF_UP
+from typing import Iterable
 
 RATE_SOURCE_ARTICLE = "статья 665 Налогового кодекса РК (Кодекс РК № 214-VIII)"
```

Ruff (0.16.1)

[warning] String contains ambiguous `Р` (CYRILLIC CAPITAL LETTER ER). Did you mean `P` (LATIN CAPITAL LETTER P)?

(RUF001)